packages/dds/map/src/directory.ts

```diff
@@ -1744,7 +1873,7 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
     /**
      * Clear all keys in memory in response to a remote clear, but retain keys we have modified but not yet been ack'd.
      */
-    private clearExceptPendingKeys() {
+    private clearExceptPendingKeys(local: boolean) {
```

Comment:
What is "local" used for?

Reply:
It is used in clearCore to send that information in the emit but I didn't send it, (is set to false rn) which is a bug. I will change it thank you